src/material-examples/table-http/table-http-example.ts

```diff
@@ -38,7 +38,7 @@ export class TableHttpExample implements OnInit {
         switchMap(() => {
           this.isLoadingResults = true;
           return this.exampleDatabase!.getRepoIssues(
-            this.sort.active, this.sort.direction, this.paginator.pageIndex);
+            this.sort.active as string, this.sort.direction as string, this.paginator.pageIndex);
```

Comment:
This shows the breaking change - as this.sort.active can be something else than just string it required casting to string. I don't really see any way around it other than having different properties for multi sort, but it would introduce more chaos.